react_components/index.js

```diff
@@ -25,8 +35,18 @@ export default function cardstrap(containerType) {
         console.info(`New Event ${e.type} for ${targetElement}`);
         console.info(targetData);
 
+        const messages = {
```

Comment:
rename this to translations and move its definition out to right after the const frJson declaration